kolibri/content/models.py

```diff
@@ -302,5 +303,7 @@ class ChannelMetadataCache(ChannelMetadataAbstractBase):
     This class stores the channel metadata cached/denormed into the primary database.
     """
 
+    last_updated = models.DateTimeField(null=True)
```

Comment:
I just merged my PR for `DateTimeTZField`, after a brief discussion with @indirectlylit and @jamalex seems like a good idea to use this in most places.

`from kolibri.core.fields import DateTimeTZField`